appveyor.yml

```diff
@@ -99,6 +99,7 @@ test_script:
       --flag cardano-sl:with-wallet
       --flag cardano-sl-core:-asserts
       --flag cardano-sl-core:-dev-mode
+      --flag cardano-sl:bench
```

Comment:
Genuine question: how come you choose to kept building the benches on Windows? Isn't less build time always a good time for CI? 😉